src/components/RightPane/RightPaneRoot.tsx

```diff
@@ -7,9 +7,10 @@ import AddedCoursePane from './AddedCourses/AddedCoursePane';
 import CoursePane from './CoursePane/CoursePaneRoot';
 import darkModeLoadingGif from './CoursePane/SearchForm/Gifs/dark-loading.gif';
 import loadingGif from './CoursePane/SearchForm/Gifs/loading.gif';
-import RightPaneStore from './RightPaneStore';
+import UCIMap from './Map/UCIMap';
+import RightPaneStore, { BuildingFocusInfo } from './RightPaneStore';
+// const UCIMap = React.lazy(() => import('./Map/UCIMap'));
```

Comment:
Why is the lazy import not working any more? Also, if we really don't need it then you can just delete it instead of commenting the line out.